cpu/fe310/cpu.c

```diff
@@ -110,8 +112,13 @@ void cpu_init(void)
     /* Initialize IRQs */
     irq_init();
 
+#ifdef _PICOLIBC__
+    /* Initialize stdio */
+    stdio_init();
```

Comment:
We should probably just remove that from `nanostubs_init()` so we can drop the `ifdef` here.

Reply:
I left the call to setvbuf (which picolibc doesn't have).